<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt 串行通讯</string>
  </property>
  <widget class="QGroupBox" name="grpComm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>串口设置</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>36</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>36</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>串口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboPort">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cboBaud">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11520</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnOpenCom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCloseCom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭串口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboFlow">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>0</height>
     </size>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>硬件</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>软件</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>36</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>流控制</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpRecv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>290</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>接收数据</string>
   </property>
   <widget class="QTextEdit" name="txtRecv">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>270</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192,192,192);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpSend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>290</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>发送数据</string>
   </property>
   <widget class="QTextEdit" name="txtSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>40</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cboPort</tabstop>
  <tabstop>cboBaud</tabstop>
  <tabstop>cboFlow</tabstop>
  <tabstop>btnOpenCom</tabstop>
  <tabstop>btnCloseCom</tabstop>
  <tabstop>txtRecv</tabstop>
  <tabstop>txtSend</tabstop>
  <tabstop>btnSend</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
